pkg/storaged/stratis-panel.jsx

```diff
@@ -80,17 +80,17 @@
         return client.stratis_pools[path_a].Name.localeCompare(client.stratis_pools[path_b].Name);
     }
 
-    function cmp_locked_pool(uuid_a, uuid_b) {
+    function cmp_stopped_pool(uuid_a, uuid_b) {
         return uuid_a.localeCompare(uuid_b);
     }
 
     const pools = Object.keys(client.stratis_pools).sort(cmp_pool)
             .map(p => stratis_pool_row(client, p));
 
-    const locked_pools = Object.keys(client.stratis_manager.LockedPools).sort(cmp_locked_pool)
-            .map(uuid => stratis_locked_pool_row(client, uuid));
+    const stopped_pools = Object.keys(client.stratis_manager.StoppedPools).sort(cmp_stopped_pool)
+            .map(uuid => stratis_stopped_pool_row(client, uuid));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230420-131050-f1115ac6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)